### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by ShahierNashaat**
Pull request for reviewing the accessibility of my portfolio

Fixing Heading

Updating alt text in images

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @ShahierNashaat, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -71,9 +71,9 @@ <h2>My Recent Works</h2>
 
             <div class="cards">
                 <div class="card">
-                    <img src="images/Img-Plaholder.png" alt="project pricture">
+                    <img src="images/Img-Plaholder.png" alt="Multi-Post Stories Gain+Glory project image">
```
Comment (mke2111) [inline index.html:74]: - [ ] For all the alt text, it is always better not to include direct description of the image such "project image". remove these kind of direct descriptions from your alt texts.

File: index.html [hunk 2]
```diff
@@ -14,7 +14,7 @@
     <meta charset="UTF-8">
```
Comment (SajjadAhmad14) [inline index.html:14]: [ OPTIONAL ]
The report should include a check about page titles.
The report should include a check about image text alternatives (img alt="alt text ").
The report should include a check about text headings.
The report should include a check about color contrast.
The report should include a check about resizing.
The report should include a check about interaction.
The report should include a check about moving content.
The report should include a check about multimedia.
The report should include a check about the basic structure of the page.

Please include a statement about every point checked in the result section of the report as described below:

If everything was ok, please make a statement about that (e.g. "Page titles - no issues found").
If issues were found, please provide a link to the corresponding line of code in the Pull Request which you change to follow the accessibility rule.

Please go through the requirements of this milestone from [this](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/6_accessability.md) link.

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Chimwemwe and Shahier, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏
# OPTIONAL BUT RECOMMENDED
- [ ] Please write a detailed accessibility report of your project and add it to the description of the pull request.
## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**